examples/models/llama/rope.py

```diff
@@ -209,18 +210,67 @@ def hf_apply_rotary_emb_to_k(k, cos, sin, position_ids=None, unsqueeze_dim=1):
     return k_embed
 
 
+# ======================= Qwen2 Implementation ========================
```

Comment:
If we put existing llama_transformer implementation, and Qwen2 implementation, side by side, what are the differences?

Reply:
Ah I meant to remove this.